CONTRIBUTING.md

```diff
@@ -19,6 +19,15 @@ Here are a few things you can do that will increase the likelihood of your pull
 * Fork the repository on GitHub.
 * Try to keep your local repository in a "rebased" state.
 
+## Finding things to work on
+
+Issues marked with [good first issue][good-first-issue] or [help wanted][help-wanted] are usually pretty self-contained and a good place to get started.
+
+If you see any issues that are assigned to a particular person or have the `work in progress` label, that means
```

Comment:
Let's write the same labels as are defined in the OSPO by replacing `work in progress` with `📍 Work in progress`!